src/main/java/org/cbioportal/legacy/web/ReferenceGenomeGeneController.java

```diff
@@ -60,11 +60,7 @@ public ResponseEntity<List<ReferenceGenomeGene>> getAllReferenceGenomeGenes(
       @Parameter(required = true, description = "Name of Reference Genome hg19") @PathVariable
           String genomeName) {
 
-    List<ReferenceGenomeGene> genes = geneMemoizerService.fetchGenes(genomeName);
-    if (genes == null) {
-      genes = referenceGenomeGeneService.fetchAllReferenceGenomeGenes(genomeName);
-      geneMemoizerService.cacheGenes(genes, genomeName);
```

Comment:
This is the only place where we use the gene memoizer service. We should probably remove the interface and the implementation class as well.